src/AtelierTomato.Markov.Storage.Sqlite/AuthorGroupPermissionRow.cs

```diff
@@ -0,0 +1,34 @@
+﻿using AtelierTomato.Markov.Model;
+
+namespace AtelierTomato.Markov.Storage.Sqlite
+{
+	public class AuthorGroupPermissionRow
+	{
+		public string ID { get; set; }
+		public string Author { get; set; }
+		public string Permissions { get; set; }
+		public AuthorGroupPermissionRow(string ID, string author, string permissions)
+		{
+			this.ID = ID;
+			Author = author;
+			Permissions = permissions;
+		}
+		public AuthorGroupPermissionRow(AuthorGroupPermission authorGroupPermission)
+		{
+			ID = authorGroupPermission.ID.ToString();
+			Author = authorGroupPermission.Author.ToString();
+			Permissions = authorGroupPermission.Permissions.ToString();
+		}
+		public AuthorGroupPermission ToAuthorGroupPermission()
+		{
+			if (!Enum.TryParse(Permissions, out AuthorGroupPermissionType permissions))

```

Comment:
you should test if tryparse on a flags enum does what it should since i am honestly not sure if it does (add unit test for this method)